src/display/components/total_bandwidth.rs

```diff
@@ -20,10 +20,12 @@ impl<'a> TotalBandwidth<'a> {
             } else {
                 Color::Green
             };
+            let interface = datalink::interfaces().into_iter().map(|interface|interface.name).collect::<Vec<String>>();
 
             [Text::styled(
                 format!(
-                    " Total Rate Up / Down: {} / {} {}",
+                    " Interfaces {:?} / Total Rate Up / Down: {} / {} {}",
```

Comment:
I'm a little concerned that with a lot of interfaces, this line might become rather long.
Since we can only listen on one interface or on all of them, what do you think about specifying either the one interface name, or something like "*" (or ALL?) if there is more than one?

Reply:
@imsnif There was a mistake to import network interface. There is no vector, only String<Option>.
Could you please take a look again?